quinn-proto/src/config.rs

```diff
@@ -444,8 +445,8 @@ where
     }
 
     /// Private key used to authenticate data included in handshake tokens.
-    pub fn token_key(&mut self, value: &[u8]) -> Result<&mut Self, ConfigError> {
-        self.token_key = Arc::new(S::HmacKey::new(value)?);
+    pub fn token_key(&mut self, master_key: &[u8]) -> Result<&mut Self, ConfigError> {
```

Comment:
`master_key` here is just constrained to `&[u8]` but actually we expect `64` bytes.

Reply:
Does this actually have to be 64 bytes? [The underlying method's docs](https://docs.rs/ring/0.16.9/ring/hkdf/struct.Salt.html#method.extract) don't seem to indicate any particular input size, even in the cited RFC.

Reply:
You are right, there is no reason for the `master_key` to be `64` bytes, I am not sure why 64 was stuck in my head.
Seems like this implementation is sufficient then :+1: